src/main/java/com/redhat/devtools/lsp4ij/features/completion/LSPCompletionContributor.java

```diff
@@ -151,7 +151,8 @@ private void addCompletionItems(@NotNull PsiFile file,
             } else {
                 // Should happen rarely, only when text edit is for multi-lines or if completion is triggered outside the text edit range.
                 // Add the IJ completion item (lookup item) which will use the IJ prefix
-                result.addElement(prioritizedLookupItem);
+                result.caseInsensitive() // set case-insensitive to search Java class which starts with upper case
```

Comment:
remove mentionof Java classes, here and above. lsp4ij completion is language agnostic